terraform/env_types/gcp/gke/k8s_setup/main.tf

```diff
@@ -39,30 +39,18 @@ module "dashboard" {
   tls_secret_crt       = module.base_setup.tls_secret_crt
 }
 
-module "dex" {
```

Comment:
You did not delete the entire dex terraform module.

Reply:
Done